src/reducers/localeFactory.js

```diff
@@ -2,17 +2,20 @@ import Immutable from "immutable";
 import { CHANGE_LOCALE, GET_CULTURES_SUCCESS } from "../actions/locale";
 
 const localeFactory = supportedLocales => {
+	const defaultCulture =
```

Comment:
@gertsonderby Do we have some app that uses 'supportedLocales' in the old format(array of tags) ?

Reply:
Analytics and OMS will need to be changed accordingly when updating to the next version of orc-shared. This is in my to do list for OMS. Analytics might be handled by Gert himself.

Reply:
This change needs to be made when updating the app to use the versions of orc-scripts and orc-shared that contain this.